src/seedsigner/views/seed_views.py

```diff
@@ -2294,3 +2301,289 @@ def run(self):
 
         # Exiting/Canceling the QR display screen always returns Home
         return Destination(MainMenuView, skip_current_view=True)
+
+
+
+"""****************************************************************************
+    Shamir's Secret Sharing Views
+****************************************************************************"""
+class SeedEntryShamirThresholdView(View):
+    def __init__(self):
+        super().__init__()
+
+
+    def run(self):
+        title = _("SLIP-39 Threshold")
+        ret_dict = self.run_screen(seed_screens.SeedEntryShamirThresholdScreen, entered_number="", title=title)
+
+        if "is_back_button" in ret_dict:
+            return Destination(BackStackView)
+            
+        elif ret_dict["entered_number"] != "" and ret_dict["entered_number"] != "0":
+            return Destination(SeedShamirShareImportSelectWordCount, view_args={"k": int(ret_dict["entered_number"])})
+
+        else:
+            if ret_dict["entered_number"] == "0":
+                from seedsigner.gui.toast import ErrorToast
+                self.controller.activate_toast(ErrorToast(_("Threshold not valid")))
+
+        return Destination(BackStackView)
+        
+
+
+class SeedShamirShareImportSelectWordCount(View):
+    TYPE_12WORD = ButtonOption("20 words (128-bit entropy)")
+    TYPE_24WORD = ButtonOption("33 words (256-bit entropy)")
```

Comment:
I had to do a bit of research here to understand. So we're talking about the words per Shamir share (20 or 33) and those, respectively, correspond to a 128- or 256-bit seed, which ultimately corresponds to a 12- or 24-word final mnemonic.

So I would omit the parentheses entirely; the words per share are all that matter.